app/Http/Middleware/LogginMiddleware.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Log;
+
+class LoggingMiddleware
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $url = $request->fullUrl();
+        $method = $request->method();
+        $ip = $request->ip();
+        Log::info("$method: $url - $ip");
+
+        return $next($request);
+    }
+}
```

Comment:
Do you actually need this LoggingMiddleware from the shop-project? If so you need to also register the middleware otherwise it won't be called.